.circleci/config.yml

```diff
@@ -86,6 +86,16 @@ jobs:
           command: |
             coverage run  --source=. -m unittest discover tests
             bash <(curl -s https://codecov.io/bash) -cF python
+  test-js:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: yarn
+      - run:
+          name: Test JS
+          command: yarn test-js
 workflows:
   version: 2
   build:
```

Comment:
To get this running in CI you need to add `- test-js` to the `jobs:` list below.

Reply:
done